atomic_reactor/plugins/pre_resolve_composes.py

```diff
@@ -287,7 +287,19 @@ def wait_for_composes(self):
             compose_info = self.odcs_client.wait_for_compose(compose_id)
 
             if self._needs_renewal(compose_info):
-                compose_info = self.odcs_client.renew_compose(compose_id)
+                sigkeys = compose_info.get('sigkeys', '').split()
+                updated_signing_intent = self.odcs_config.get_signing_intent_by_keys(sigkeys)
+                if set(sigkeys) != set(updated_signing_intent['keys']):
+                    self.log.info('Updating signing keys in "%s" from "%s", to "%s" in compose '
+                                  '"%s" due to sigkeys deprecation',
+                                  updated_signing_intent['name'],
+                                  sigkeys,
+                                  updated_signing_intent['keys'],
+                                  compose_info['id']
+                                  )
+                    sigkeys = updated_signing_intent['keys']
+
+                compose_info = self.odcs_client.renew_compose(compose_id, sigkeys)
```

Comment:
this should also handle old odcs which doesn't accept sigkeys

Reply:
It does already. ODCS will just ignore the sigkey param if it is provided to old ODCS instances.

Reply:
ahh ok